### ISSUE TITLE
Validation errors should be printed in multiline format by kubectl

### ISSUE THREAD

**Issue by smarterclayton** [GoogleCloudPlatform/kubernetes#7027]
422 Invalid should result in an error message in kubectl like:

```
Error: the Pod "foo" is invalid:
* name is a required value
* something else is wrong
```

`cmd/util/helpers.go#CheckErr` should handle the display of an APIStatus with Causes more effectively

**Comment by andronat** [comment]
Hello, is this still open? Because in this commit: a47716e66dc9c1f8662935f6cbbe879cc869d548 I see that multiple errors can be printed.

**Comment by smarterclayton** [comment]
We need to specifically check and handle the IsInvalid condition and get access to the individual errors returned by the response (Ststus.details.causes)

**Comment by andronat** [comment]
I think this can be closed now

**Comment by andronat** [comment]
I think this can be closed now

**Comment by smarterclayton** [comment]
Fixed by #9130
